wpiutil/BUILD.bazel

```
@@ -209,7 +228,10 @@ cc_library(
 cc_test(
     name = "wpiutil-cpp-test",
     size = "small",
-    srcs = glob(["src/test/native/cpp/**"]),
+    srcs = glob([
+        "src/test/native/cpp/**",
+        "src/generated/test/native/cpp/*.cpp",
```

I suspect this line is preventing the generated nanopb header from being included as part of the build and causing the build to fail.
```suggestion
        "src/generated/test/native/cpp/*",
```